test/packages/good/elasticsearch/transform/metadata_current/transform.json

```diff
@@ -0,0 +1,32 @@
+{
+    "source": {
+        "index": "metrics-endpoint.metadata-*",
+        "query": {
+            "range": {
+                "@timestamp": {
+                    "gt": "now-90d/d"
+                }
+            }
+        }
+    },
+    "dest": {
+        "index": "metrics-endpoint.metadata_current_default"
+    },
+    "latest": {
+        "unique_key": [
+            "elastic.agent.id"
+        ],
+        "sort": "@timestamp"
+    },
+    "description": "Latest Endpoint metadata document per host",
+    "_meta": {
+        "managed": true
+    },
+    "frequency": "1s",
+    "sync": {
+        "time": {
+            "field": "event.ingested",
+            "delay": "1s"
+        }
+    }
+}
```

Comment:
Nit. We support defining pipelines as YAML, that are converted to JSON on install time. This makes them easier to maintain.
Would it make sense to do the same here?

Reply:
Yeah I think it could work here as well

Reply:
changed to yml.